docs/specs/om/open_metrics_spec_1_1.md

```diff
@@ -633,7 +635,9 @@ foo 18.0 456
 
 ##### Counter
 
-The MetricPoint's Total Value Sample MetricName MUST have the suffix `_total`. If present the MetricPoint's Created Value Sample MetricName MUST have the suffix `_created`.
+The MetricPoint's Total Value Sample MetricName SHOULD have the suffix `_total`. If present the MetricPoint's Created Value Sample MetricName MUST have the suffix `_created`.
```

Comment:
hmmm, I'm not sure how Otel deals with created metrics.  I am tempted not to have any MUSTs around names so that we can truly allow anything and not be against the spec (no matter how much we discourage it)

Reply:
I think we need a MUST for the _created series, otherwise it wouldn't be the `_created` series :).